modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -1088,6 +1088,24 @@ configuration.
 - [LPS-70819]: Publish a JAR file with the compiled JSP classes of an OSGi
 project with the `install` and `uploadArchives` tasks.
 
+### Changed
+- [LPS-66853]: Update the [Liferay Gradle Plugins] dependency to version 3.2.7.
+
+## 3.0.0 - 2017-02-22
+
+### Changed
+- [LPS-63943]: Refactor and rename `PrintArtifactPublishCommandsTask` class to
+`WriteArtifactPublishCommandsTask`. Executing
+`gradlew writeArtifactPublishCommands` in a parent directory causes the file
+`build/artifacts-publish-commands/artifacts-publish-commands.sh` to be generated
+with the following commands for all publishable subprojects:
+	- `gradlew baseline`
+	- `gradlew baseline` and Git commands to commit the files modified by the
+	`baseline` task
```

Comment:
They are 2 distinct points:
1. `gradlew baseline` (failing in case of semantic versioning errors)
2. `gradlew baseline` (ignoring semantic versioning errors) and Git command, etc.